dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/handlers/ResponseFallbacks.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.dimensiondata.cloudcontrol.handlers;
+
+import com.google.common.base.Throwables;
+import org.jclouds.Fallback;
+import org.jclouds.rest.ResourceNotFoundException;
+
+public final class ResponseFallbacks {
+   private ResponseFallbacks() {
+   }
+
+   public static final class NullOnResourceNotFound implements Fallback<Object> {
+      public Void createOrPropagate(Throwable t) {
+         if (t instanceof ResourceNotFoundException) {
+            return null;
+         } else {
+            throw Throwables.propagate(t);
+         }
+      }
+   }
+
+   public static final class VoidOnResourceNotFound implements Fallback<Void> {
+      public Void createOrPropagate(Throwable t) {
+         if (t instanceof ResourceNotFoundException) {
+            return null;
+         } else {
+            throw Throwables.propagate(t);
+         }
+      }
+   }
```

Comment:
You'd better copy the [default behavior](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/Fallbacks.java#L132) but changing the code to 400, to make sure you just capture the right responses.

Reply:
I see now that NullOnNotFoundOr404 / VoidOnNotFoundOr404 might do the job here (they also trigger on ResourceNotFoundException). Let me check, maybe I can revert to using them.